src/selection.ts

```diff
@@ -0,0 +1,62 @@
+export enum SelectionTypes {
```

Comment:
> "renaming symbol" option in VSCode doesn't seem to propagate changes

AFAIK, I think you still need to rely on find and replace for renaming.  
That said, the typing will ensure that you don't forget to rename them if you rename the value in the type first.  

By this I mean, if you change
```
export type SelectionType = 'single' | 'multi' | 'interval';
```
to
```
export type SelectionType = 'point' | 'list' | 'interval';
```
first.  The build will keep failing until you have replaced all single/multi SelectionTypes.

> would I not need to still import SelectionType in b.ts to be able to designate it as the type of a? 

Yes.  So you still need to import to designate type, but no longer need to import to designate value.  

For example, for declaring variable and function types you still need to import:

```
import {ScaleType} from './scale';
const scaleType: ScaleType = 'linear';  
export function a(scaleType: ScaleType);
```

If you import `a()` to another file, you can just call

```
a('linear');
```

as opposed to 

```
import {ScaleType} from './scale';
a(ScaleType.LINEAR); 
```

That said, if both you and @domoritz prefer the latter (declare a namespace + type like the current ScaleType code) rather than just the type like I mention above, I'm okay with that too.